kubeflow/trainer/rhai/traininghub.py

```
@@ -38,33 +55,409 @@ class TrainingHubTrainer:
+                    int(remaining_steps * time_per_batch) if time_per_batch > 0 else None
+                )
+
+            loss_val = metrics.get("loss", 0)
+            lr_val = metrics.get("lr")
+            grad_norm_val = metrics.get("grad_norm", 0)
+            metrics.get("samples_per_second", 0)
+            val_loss_val = metrics.get("val_loss")
+
+            return {
+                {
+                    "progressPercentage": int(round(percent)),
```

Ruff (0.14.7)

Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)